src/features/Main/Header/index.scss

```diff
@@ -27,7 +27,7 @@ header.institution-header {
   }
 
   .header-left img {
-    max-width: 30px;
+    height: 1.8rem;
```

Comment:
Why this change?

Reply:
the width can change if logo is longer or not, but the height should be fixed to avoid be bigger than the other elements